packages/cmcd-validator-library/src/tests/testCases/queryTestCases.js

```diff
@@ -1327,4 +1327,71 @@ export const queryTestCases = [
       rawData: 'https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps_3840x2160_12000k/bbb_30fps_3840x2160_12000k_0.m4v?CMCD=cid%3D%2221cf726cfe3d937b5f974f72bb5bd06a%22%2Cot%3Di%2Csf%3Dd%2Cst%3Dv%2Csu',
     },
   },
+  {
+    description: 'Test: Query malformed',
+    query:
+    'https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps_3840x2160_12000k/bbb_30fps_3840x2160_12000k_0.m4v?sdfsdfCMCD=cid%3D%2221cf726cfe3d937b5f974f72bb5bd06a%22%2Cot%3Di%2Csf%3Dd%2Csid%3D%22b248658d-1d1a-4039-91d0-8c08ba597da5%22%2Cst%3Dv%2Csu%0x26qualabs',
+    output:
+    {
+      valid: false,
+      errors: [
+        {
+          type: 'query-malformed',
+          key: undefined,
+          value: undefined,
+          description: 'The query is malformed'
+        }
+      ],
+      warnings: [],
+      parsedData: undefined,
+      rawData: 'https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps_3840x2160_12000k/bbb_30fps_3840x2160_12000k_0.m4v?sdfsdfCMCD=cid%3D%2221cf726cfe3d937b5f974f72bb5bd06a%22%2Cot%3Di%2Csf%3Dd%2Csid%3D%22b248658d-1d1a-4039-91d0-8c08ba597da5%22%2Cst%3Dv%2Csu%0x26qualabs'
+    },
+  },
+  {
+    description: 'Test: If the request already bears a query string, checks if there is a "&" before CMCD query #1',
+    query:
+    'https://dash.akamaized.net/akamai/bbb_30fps/bbb_30fps_3840x2160_12000k/bbb_30fps_3840x2160_12000k_0.m4v?qualabsCMCD=cid%3D%2221cf726cfe3d937b5f974f72bb5bd06a%22%2Cot%3Di%2Csf%3Dd%2Csid%3D%22b248658d-1d1a-4039-91d0-8c08ba597da5%22%2Cst%3Dv%2Csu&hola',
+    output:
+    {
+      valid: false,
+      errors: [
+        {
+          type: 'no-ampersand-between-requests:',
+          key: undefined,
+          value: undefined,
+          description: 'Ampersand required between two or more requests'
+        }
+      ],
+      warnings: [],
```

Comment:
The parsedData field should be undefined, as there is an error in the quertValidator function and it is not supposed to parse the data in this case.

Reply:
that's right! I'm going to change it right now, thanks!